lib/replicant/shared.js

```diff
@@ -20,6 +20,12 @@ const ARRAY_MUTATOR_METHODS = [
 	'unshift'
 ];
 
+/**
+ * The default persistence interval, in milliseconds.
+ * @type {Number}
+ */
+const DEFAULT_PERSISTENCE_INTERVAL = 250;
```

Comment:
250ms (= 4 times a second) feels slower as default value, but would like to hear @Lange's opinion

Reply:
Hm yeah, we know that 60hz (~16.67 ms) is too short an interval, because we've seen many Replicants encounter errors when persisting at that high of a rate. I think 100ms is probably a nice compromise that is likely to be successful most of the time?